cea/demand/ventilation_model.py

```diff
@@ -565,7 +565,7 @@ def allocate_default_ventilation_openings(coeff_vent_zone, height_zone):
     # initialize
     coeff_vent_path = np.zeros(4)
     height_vent_path = np.zeros(4)
-    orientation_vent_path = np.zeros(4)
+    orientation_vent_path = [0, 1, 0, 1]
```

Comment:
No. Right now it does not have any implications. As windows are not used anyways in any calculation. As far as I understand.
Why would you change that?
And what does this have to do with cooling coils?